tardis/apps/yaml_dump/utils/experiment.py

```diff
@@ -0,0 +1,65 @@
+"""utils.experiment.py
+
+A set of functions to wrangle the data into an appropriate format for export via YAML
+
+Author: Chris Seal <c.seal@auckland.ac.nz>
+"""
+import contextlib
+from typing import Any, Dict
+
+from django.conf import settings
+
+from tardis.apps.yaml_dump.models.experiment import Experiment
+from tardis.apps.yaml_dump.utils.utility import (
+    add_acls_to_dataclass,
+    add_data_classification_to_dataclass,
+    add_metadata_to_dataclass,
+)
+
+INGESTED = 5
```

Comment:
Is there a reason for using a constant for the INGESTED data status value, instead of
[tardis/apps/yaml_dump/models/datastatus.py](https://github.com/UoA-eResearch/mytardis/pull/466/files#diff-a1a9acbc4050b91d86336a46a921624401e3a3ce524aa140d65fc8205a0921ec)?

Reply:
Partly for simplicity, partly because the YAML dump tries to add tags for objects that are 'non-standard' so I ended up with a tag something like !!tardis.apps.yaml_dump.models.datastatus.DataStatus